supplierinfo_product_breakdown/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows to adding supplier product components
```

Comment:
This module allows to relate components and their qty to a product supplierinfo for a specific vendor.

It can be used as a base module for further implementations.